Directory.Build.targets

```diff
@@ -1,10 +1,18 @@
 <Project>
-
+  <!-- Set code coverage properties that reference properties not available in Directory.Build.props -->
+  <PropertyGroup Condition="'$(Coverage)' == 'true'">
+    <CoverletOutput>$(TargetDir)coverage\$(MSBuildProjectName).coverage</CoverletOutput>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Coverage)' == 'true'">
```

Comment:
This and the above PropertyGroup use the same Condition so they can be combined into one.

Reply:
How about creating a Testing.props and putting these PropertyGroup's there ?

Reply:
Sure. Combining and putting in Testing.targets.